src/app/content/components/Page/contentDOMTransformations.ts

```diff
@@ -141,10 +122,10 @@ function wrapSolutions(rootEl: HTMLElement, intl: IntlShape) {
     el.setAttribute('aria-expanded', 'false');
     const contents = el.innerHTML;
     el.innerHTML = `
-      <div class="ui-toggle-wrapper">
-        <button class="btn-link ui-toggle" title="${title}"></button>
-      </div>
-      <section class="ui-body" role="alert" style="${initialSolutionStyles}">${contents}</section>
+      <details class="ui-toggle-wrapper">
+        <summary class="btn-link ui-toggle" title="${title}"></summary>
```

Comment:
for intl we need to handle the contents of the summary here instead of the css we currently have, i think probably just putting the title in there would be good